scene/animation/animation_player.cpp

```diff
@@ -1430,17 +1449,17 @@ AnimationPlayer::AnimationMethodCallMode AnimationPlayer::get_method_call_mode()
 	return method_call_mode;
 }
 
-void AnimationPlayer::_set_process(bool p_process, bool p_force) {
-	if (processing == p_process && !p_force) {
+void AnimationPlayer::_set_process(bool p_process) {
+	if (processing == p_process) {
 		return;
 	}
 
 	switch (animation_process_mode) {
 		case ANIMATION_PROCESS_PHYSICS:
-			set_physics_process_internal(p_process && active);
+			set_physics_process_internal(p_process);
```

Comment:
Outside the switch I would put:
```cpp
set_physics_process_internal(false);
set_process_internal(false);
```
So, even if you change the processing mode at runtime, the processing is still correctly set.

Reply:
Changing this would be outside the scope of this PR. That said, I don't think it's necessary, because setting them to false is already done in the `set_process_callback()`:
https://github.com/godotengine/godot/blob/9b6c3a871922d63fe5b92288a373789dc7130b4c/scene/animation/animation_player.cpp#L1406-L1419